framework/sandstone_tests.h

```diff
@@ -23,6 +23,18 @@ extern struct test __stop_tests;
 
 extern struct test mce_test;
 
+typedef enum {
+    TEST_ENABLED,
+    TEST_DISABLED,
+    TEST_NOT_FOUND,
+} test_cfg_status;
```

Comment:
```c++
enum class test_cfg_status {
    disabled = 0,
    enabled = 1,
    not_found = -1,
};
```
The `class` forces you to use scoping: `test_cfg_status::enabled`. Or move it inside the `struct tst_cfg_info` struct below.